### DISCUSSION TITLE
Great tool, but does it handle conflicts?

### DISCUSSION THREAD

**Discussion by jaypoc** [software-2/LightroomSync#5] [Q&A] [answered]
I love this, and plan to give it a try. I'm curious how you're managing conflicts. Say you leave your desktop PC open to LR but start an editing session on your Laptop? When you close each of them, they will have different states. Do you get a notification when opening LR that another PC has the library open? or are you performing any type of diff and merge of the libraries?

**Comment by software-2** [comment, marked as answer]
The app forcibly closes Lightroom if it detects there's another instance of Lightroom open.
https://github.com/software-2/LightroomSync/blob/08a7fd9a1ede0073800edf5f62998ab746a07c60/LightroomSync/Form1.cs#L447

**Reply by jaypoc** [reply]
Thank you.
